doc/cf_deployment_bump.md

```diff
@@ -0,0 +1,20 @@
+# Bump
```

Comment:
This file makes sense to put under `doc/cf_deployment/bump.md`.

Reply:
Fixed with commit 50709e88bd8c8904bda167df86e25e06e621d083